src/components/GalleryGrid.tsx

```diff
@@ -1,60 +1,74 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import GalleryCard from "./GalleryCard";
 import { GalleryCardType } from "./GalleryCard";
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const staticImage: GalleryCardType = {
   title: "The Bedroom",
-  artist: "Vincent van Gogh",
-  location: "Saint-Rémy-de-Provence, 1889",
+  artist_title: "Vincent van Gogh",
+  date_display: "Saint-Rémy-de-Provence, 1889",
+  place_of_origin: "something",
   image:
     "https://upload.wikimedia.org/wikipedia/commons/0/0b/Vincent_van_Gogh_-_The_Bedroom_-_Google_Art_Project.jpg",
-  altText: "Gustav Klimt painting",
+  image_id: "10",
 };
 
 function GalleryGrid() {
-  const [galleries, setGalleries] = useState<GalleryCardType[]>([]);
-  for (let i = 0; i < 10; i++) {
-    galleries.push(staticImage);
-  }
+  const [artworks, setArtworks] = useState<GalleryCardType[]>([]);
+  useEffect(() => {
+    const fetchArtworks = async () => {
+      try {
+        const response = await fetch(
+          "https://api.artic.edu/api/v1/artworks?page=1&fields=title,artist_title,place_of_origin,date_display,image_id"
+        );
+
```

Comment:
I am showwing all page 1 entries but I can easily change the api call and add limit=10 once im back, just noticed that in the A.Cs

Reply:
I've set the limit to 10 now